universal/router/Root/index.js

```diff
@@ -0,0 +1,14 @@
+/* @flow */
+import React from 'react'
+
+import Home from 'layouts/Home'
+
+const HomeRoute = () => (
+  <Home />
+)
```

Comment:
Is there benefit here to creating this HomeRoute component? Looks like we could just pass in the home component to the export. I'm not sure i completely grok this directory pattern...